Jenkinsfile

```diff
@@ -7,10 +7,11 @@ pipeline {
     }
     parameters {
         choice(name: "CHANNEL", choices: ["nightly", "dev", "beta", "release"], description: "")
+        choice(name: "BUILD_TYPE", choices: ["Release", "Debug"], description: "")
```

Comment:
How does default value get set? (asked same question in https://github.com/brave/brave-core/pull/2443/files#r289559506). If I check in `brave-browser`, how does it know which build type to default to?

Reply:
first in the list is default